myplanet/skeletor
# SKEL-32: Remove language dependency

## Testing notes
* Checkout branch
* Create new drupal site install
* Enable all the Skeletor components (with drupal console: `drupal moi skeletor_accordion skeletor_hero skeletor_media skeletor_media_object skeletor_slideshow skeletor_tabs_component skeletor_text_component -y`)
* Language module should not be installed as a dependency

JIRA Story: https://myplanet.jira.com/browse/SKEL-32
